assets/test/templates/common/function.tmpl

```diff
@@ -118,8 +118,9 @@ func {{ .TestName }}(t *testing.T) {
                 if test.afterFunc != nil {
                     defer test.afterFunc({{- if .TestParameters }} test.args {{- end }})
                 }
+                checkFunc := test.checkFunc
```

Comment:
could you please the reason why you changed?

Reply:
It is the fix of the race error in test template.
When the test is executed in parallel mode, changing the objects inside the slice `tests` in multiple goroutine will cause race error.
To avoid the race error, store the `checkFunc` as local variable to avoid modifying `test` object or the `tests` slice.

Reply:
Ok